src/Controller/Api/SrpController.php

```diff
@@ -242,4 +244,56 @@ public function loginAction(Request $request, SrpUserManager $srpUserManager, JW
             'jwt' => $jwtManager->create($loginRequest->getUser()),
         ];
     }
+
+    /**
+     * @SWG\Tag(name="Srp")
+     *
+     * @SWG\Parameter(
+     *     name="body",
+     *     in="body",
+     *     @Model(type=\App\Form\Request\Srp\UpdatePasswordType::class)
+     * )
+     * @SWG\Response(
+     *     response=204,
+     *     description="Password changed"
+     * )
+     *
+     * @SWG\Response(
+     *     response=403,
+     *     description="Access denied"
+     * )
+     *
+     * @Route(
+     *     path="/api/srp/password",
```

Comment:
TODO: All authorization endpoints must be on the same path. 
For ex.: `/api/auth/oauth` or `/api/auth/srpp`

P.S. SRPP instead of SRP, because we show that is authorization protocol like OAuth.